source/includes/_sequences.md

```diff
@@ -122,20 +141,352 @@ padding      | body    | Max number of digits the code entries will have (they w
 
 ## Retrieve a code
 
-Something here
+```shell
+curl "https://api.invopop.dev/sequence/:owner_id/code/:code_id" \
+  -H "Authorization: Bearer your.api.key"
+```
+
+```go
+import (
+    "bytes"
+    "encoding/json"
+    "net/http"
+)
+
+type Code struct {
+    ID      string `json:"id"`
+    Name    string `json:"name"`
+    Preffix string `json:"preffix"`
+    Suffix  string `json:"suffix"`
+    Padding int    `json:"padding"`
+}
+
+httpClient := &http.Client{}
+req, _ := http.NewRequest("GET", "https://api.invopop.dev/sequence/:owner_id/code/:code_id", nil)
+req.Header.Set("Authorization", "Bearer your.api.token")
+
+res, err := client.Do(req)
+if err != nil {
+    // Deal with possible network errors here...
+}
+
+var code Code
+decodingErr := json.NewDecoder(res.Body).decode(&code)
+```
+
+> Response
+
+```json
+{
+    "id": "d4cfb49f-035b-46f0-b7b4-ec733aee2d00",
+    "name": "Some series",
+    "prefix": "XX",
+    "padding": 7,
+    "suffix": "Y"
+}
+```
+
+Retrieves a specific code object using the given UUID
+
+### HTTP Request
+
+`GET /sequence/:owner_id/code/:code_id`
+
+### Parameters
+
+Parameter    | Type    | Description
+------------ | ------- | -----------
+owner_id     | path    | Id of the company that created the code
+code_id      | path    | UUID that identifies the code
 
 ## List all codes
 
-Something here
+```shell
```

Comment:
This is probably the wrong place for the comment, but is an external user of this API going to use this operation for something? Wouldn't the user just need an operation that returns the next entry in the sequence (and maybe the current one)?

Reply:
This is indeed the debate around the public API. We should always be asking, "What is the easiest way to make this API usable?" The internal APIs via gRPC are designed to be complete and flexible, but the public APIs should be concise and easy.